resources/views/sonFormFields/edit.blade.php

```diff
@@ -0,0 +1,713 @@
+@extends('layouts.app')
+
+@section('title', 'SON Create')
+@section('page-title', 'SON Create')
+
+@section('content')
+
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header"><h3>{{ __('Statement Of Need Form Fields: ') }}</h3></div>
```

Comment:
Because it's a heading...
```suggestion
                <div class="card-header"><h3>{{ __('Statement of Need Form Fields') }}</h3></div>
```